docs/content/error/$interpolate/reqother.ngdoc

```diff
@@ -0,0 +1,13 @@
+@ngdoc error
+@name $interpolate:reqother
+@fullName Required choice "other" for select/plural in MessageFormat
+@description
+
+Your interpolation expression with a MessageFormat extension for either
+"plural" or "select" (typically used for gender selection) does not contain a
+message for the choice "other".  Using either select or plural MessageFormat
+extensions require that you provide a message for the selection "other".
```

Comment:
Yup.  You provide all the choices that you expect to match and "other" if there is no match.  For plurals, for instance, you can omit categories that you think can't occur, but you must still include "other".  For gender, you should include all the choices which might require a different message.  Since male, female and other have different messages, typically, would include all three.  However, for a message like, "You have no posts", you might include *just* other as the "You" in English is gender neutral but might not be in a different language.